tests/vision/utils_tests/metrics_test.py

```diff
@@ -12,7 +12,7 @@
 
 from deepchecks.vision import VisionData
 from deepchecks.vision.metrics_utils.metrics import calculate_metrics
-from deepchecks.vision.metrics_utils.object_detection_precision_recall import ObjectDetectionAveragePrecision
+from deepchecks.vision.metrics_utils.detection_precision_recall import ObjectDetectionAveragePrecision
```

Comment:
Why did we rename?

Reply:
because it's not specifically object detection

Reply:
what other types of detection are there?